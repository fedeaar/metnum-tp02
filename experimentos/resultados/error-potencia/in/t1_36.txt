-3.610419156841547874e+02 -7.099147906790368268e+01 -6.920423921744449558e+01 -2.535798549383433098e+01 1.141487115283510434e+02 -7.664390066351812436e+01 2.185186952095631518e+00 2.558622500072142358e+01 -1.478254828526459397e+00 3.171196642374930619e+01 2.827930182759023303e+00 -4.117030521443195568e+01 1.179790185531664548e+01 8.014019662175208225e+01 4.700860841725202022e+01 5.714729238270228251e+01 -3.026369987991659016e+00 6.632339496042747840e+01 5.149188724078904045e+01 2.060427477035045918e+00
-7.099147906790368268e+01 6.530172767943665804e+02 -1.707759504804897119e+02 -5.242141003512909236e+01 -3.587774232623001325e+01 -1.781730094633135764e+02 -5.836462503288647952e+01 -1.395201252659662430e+02 -1.649795937253938760e+01 -4.265240317479005228e+01 -7.783102430192376353e+00 -8.237245799422734649e+01 -1.545449301784847762e+02 -8.731828339836822295e+01 -3.674842630434936552e+01 -1.054559676314978702e+02 -5.041781687898629372e+01 -3.898561302570616505e+01 -3.423139610435875113e+01 -1.342112479833361460e+00
-6.920423921744449558e+01 -1.707759504804897119e+02 3.203751790666434545e+02 -6.447050023973989141e+01 -1.073199688767033955e+01 -2.167026220073503566e+02 -6.405582215149705405e+01 -1.479543021631863553e+02 -1.863890721950724583e+01 -3.916449378501446432e+01 -7.897950739088480887e+00 -1.016420106523030000e+02 -1.680882458515849294e+02 -7.631325086007846892e+01 -2.874447082655540342e+01 -1.022304898969203322e+02 -5.658191077484384124e+01 -2.631655255028338658e+01 -2.481965217391347878e+01 -9.625469416080614238e-01
-2.535798549383433098e+01 -5.242141003512909236e+01 -6.447050023973989141e+01 6.891198210197724165e+02 -1.072032593410221857e+01 -6.736048454680283726e+01 -2.146654195276304122e+01 -5.086872211297718138e+01 -6.113954444430229174e+00 -1.502669980611662481e+01 -2.806965050860630750e+00 -3.126780309966807536e+01 -5.670978494493496669e+01 -3.042877167406737016e+01 -1.251467639215393213e+01 -3.761930646482733209e+01 -1.865154029281664450e+01 -1.291548793350041358e+01 -1.148845714610393287e+01 -4.495161368850928429e-01
1.141487115283510576e+02 -3.587774232623001325e+01 -1.073199688767033955e+01 -1.072032593410221502e+01 -6.720242260968359460e+02 -1.921312545820713780e+01 4.295696606793469385e+01 1.394346420898406791e+02 8.360317129440314687e+00 8.573841057499043927e+01 1.030587900575725158e+01 -1.923469492745238085e+01 1.245954113859248906e+02 2.029889923806344711e+02 1.093963381175424132e+02 1.736199081918830132e+02 2.824015131002820311e+01 1.457473571192244037e+02 1.158043932620984293e+02 4.615388914881490834e+00
-7.664390066351812436e+01 -1.781730094633135764e+02 -2.167026220073503566e+02 -6.736048454680283726e+01 -1.921312545820714135e+01 3.540027367209641511e+02 -6.877770486569598063e+01 -1.602488997548989005e+02 -1.986999478609507364e+01 -4.410447114568544436e+01 -8.653065907742458762e+00 -1.061177166834504675e+02 -1.808887126367381768e+02 -8.717903452318310542e+01 -3.397418555964034681e+01 -1.133930318766572043e+02 -6.041786092106448791e+01 -3.267828417063249447e+01 -3.007210065739897331e+01 -1.170536340856643598e+00
2.185186952095626189e+00 -5.836462503288647952e+01 -6.405582215149705405e+01 -2.146654195276304122e+01 4.295696606793469385e+01 -6.877770486569598063e+01 -9.256555732368849476e+01 -1.633030884973410224e+01 -3.905393811520604164e+00 5.480649624458545865e+00 -2.969827842677531526e-01 -3.431189764425056410e+01 -2.534134717266257297e+01 1.789185629773090369e+01 1.335209187931277519e+01 4.231095233238259645e+00 -1.128118993176686935e+01 2.137762354566107703e+01 1.581443958076759770e+01 6.382664220691884882e-01
2.558622500072141648e+01 -1.395201252659662430e+02 -1.479543021631863269e+02 -5.086872211297718138e+01 1.394346420898406791e+02 -1.602488997548989005e+02 -1.633030884973409513e+01 -2.120559999541727905e+02 -7.393331898124967161e+00 2.796997099883636295e+01 1.183337629099378452e+00 -8.168700722357699817e+01 -3.527555306923444789e+01 7.742661810758026775e+01 5.018364899646478960e+01 4.098510036766130327e+01 -2.061815178032020413e+01 7.503981316852755867e+01 5.695336107005481807e+01 2.288072117681324791e+00
-1.478254828526459397e+00 -1.649795937253938760e+01 -1.863890721950724227e+01 -6.113954444430229174e+00 8.360317129440311135e+00 -1.986999478609507008e+01 -3.905393811520604608e+00 -7.393331898124968049e+00 4.369611864014122915e+01 1.878246727796414883e-02 -2.788251324030361356e-01 -9.733473391715623535e+00 -9.767691585187094461e+00 1.490285721519714279e+00 1.894143214307999257e+00 -1.981553597573590286e+00 -3.842414958218360432e+00 3.578980802981562181e+00 2.499237478912755606e+00 1.019544432230198583e-01
3.171196642374930619e+01 -4.265240317479005228e+01 -3.916449378501447143e+01 -1.502669980611662481e+01 8.573841057499041085e+01 -4.410447114568545146e+01 5.480649624458538760e+00 2.796997099883636295e+01 1.878246727796414883e-02 -4.660050886390356482e+02 2.583008476841373469e+00 -2.457899612331490857e+01 1.890200867181750155e+01 6.432989233539070995e+01 3.677136154696471948e+01 4.874804967281774992e+01 1.146244544441404001e+00 5.102373677074348990e+01 3.987745875674674778e+01 1.593840642113284245e+00
2.827930182759023303e+00 -7.783102430192377241e+00 -7.897950739088479111e+00 -2.806965050860630750e+00 1.030587900575724980e+01 -8.653065907742458762e+00 -2.969827842677538743e-01 1.183337629099378452e+00 -2.788251324030360800e-01 2.583008476841373469e+00 -3.158037618519591092e+02 -4.533831457043132396e+00 -2.274304643992269281e-01 6.703009443300687487e+00 4.055856037180579854e+00 4.409740647962161475e+00 -7.134432232727448975e-01 5.832537242656996135e+00 4.494272709196326154e+00 1.800734326241278938e-01
-4.117030521443195568e+01 -8.237245799422733228e+01 -1.016420106523030000e+02 -3.126780309966807536e+01 -1.923469492745238441e+01 -1.061177166834504533e+02 -3.431189764425055699e+01 -8.168700722357701238e+01 -9.733473391715623535e+00 -2.457899612331491213e+01 -4.533831457043132396e+00 7.018454921030863716e+02 -9.075621564914808914e+01 -5.006768869948873402e+01 -2.085260579899799893e+01 -6.112040230541991548e+01 -2.972097262509153026e+01 -2.185117002897232652e+01 -1.929750040868416150e+01 -7.559136265638379815e-01
1.179790185531661706e+01 -1.545449301784847478e+02 -1.680882458515849294e+02 -5.670978494493496669e+01 1.245954113859248906e+02 -1.808887126367382052e+02 -2.534134717266257297e+01 -3.527555306923445499e+01 -9.767691585187094461e+00 1.890200867181750155e+01 -2.274304643992277053e-01 -9.075621564914808914e+01 -1.756316937746072426e+02 5.760790876483693523e+01 4.074789337736694250e+01 2.031765876602186793e+01 -2.799713971274612945e+01 6.367319502278522236e+01 4.752883201653107648e+01 1.915135898433854011e+00
8.014019662175208225e+01 -8.731828339836820874e+01 -7.631325086007846892e+01 -3.042877167406737016e+01 2.029889923806344711e+02 -8.717903452318310542e+01 1.789185629773090369e+01 7.742661810758026775e+01 1.490285721519716944e+00 6.432989233539070995e+01 6.703009443300691039e+00 -5.006768869948872691e+01 5.760790876483694944e+01 -3.944008363125274172e+02 8.893052441489423643e+01 1.228705122449195812e+02 7.092252083280888186e+00 1.223468138564082466e+02 9.594968070241807823e+01 3.832675826531472651e+00
4.700860841725202732e+01 -3.674842630434935842e+01 -2.874447082655538921e+01 -1.251467639215393213e+01 1.093963381175424132e+02 -3.397418555964034681e+01 1.335209187931277697e+01 5.018364899646477539e+01 1.894143214307998813e+00 3.677136154696471948e+01 4.055856037180580742e+00 -2.085260579899799538e+01 4.074789337736694961e+01 8.893052441489422222e+01 -6.006596378302980384e+02 7.184567783385527662e+01 7.126125304743490929e+00 6.710275165759553317e+01 5.287065841196453420e+01 2.110202887357991663e+00
5.714729238270229672e+01 -1.054559676314978702e+02 -1.022304898969203464e+02 -3.761930646482733920e+01 1.736199081918830132e+02 -1.133930318766572043e+02 4.231095233238248987e+00 4.098510036766128906e+01 -1.981553597573591174e+00 4.874804967281773571e+01 4.409740647962162363e+00 -6.112040230541991548e+01 2.031765876602186083e+01 1.228705122449195812e+02 7.184567783385527662e+01 -3.197025843622351999e+02 -3.794924527233197953e+00 1.011631075330583514e+02 7.860296455889826461e+01 3.144831594687558685e+00
-3.026369987991663901e+00 -5.041781687898629372e+01 -5.658191077484384124e+01 -1.865154029281664450e+01 2.824015131002821022e+01 -6.041786092106448791e+01 -1.128118993176686935e+01 -2.061815178032020768e+01 -3.842414958218360432e+00 1.146244544441405777e+00 -7.134432232727448975e-01 -2.972097262509154092e+01 -2.799713971274613300e+01 7.092252083280888186e+00 7.126125304743489153e+00 -3.794924527233194400e+00 1.722546452938350070e+00 1.269030525081079297e+01 9.039994773940728479e+00 3.673976531312315164e-01
6.632339496042746418e+01 -3.898561302570617926e+01 -2.631655255028338658e+01 -1.291548793350041358e+01 1.457473571192244037e+02 -3.267828417063250868e+01 2.137762354566107348e+01 7.503981316852755867e+01 3.578980802981562181e+00 5.102373677074348990e+01 5.832537242656996135e+00 -2.185117002897232297e+01 6.367319502278520815e+01 1.223468138564082182e+02 6.710275165759553317e+01 1.011631075330583798e+02 1.269030525081079297e+01 -6.434711731281647644e+02 7.156182928114522213e+01 2.854609800849534196e+00
5.149188724078904045e+01 -3.423139610435875113e+01 -2.481965217391347878e+01 -1.148845714610393287e+01 1.158043932620984293e+02 -3.007210065739898042e+01 1.581443958076759593e+01 5.695336107005481807e+01 2.499237478912755606e+00 3.987745875674674778e+01 4.494272709196326154e+00 -1.929750040868416150e+01 4.752883201653108358e+01 9.594968070241807823e+01 5.287065841196453420e+01 7.860296455889827882e+01 9.039994773940730255e+00 7.156182928114522213e+01 -6.415066269230318312e+02 2.254043041949876702e+00
2.060427477035045918e+00 -1.342112479833361238e+00 -9.625469416080609797e-01 -4.495161368850928429e-01 4.615388914881490834e+00 -1.170536340856643598e+00 6.382664220691888213e-01 2.288072117681324791e+00 1.019544432230198860e-01 1.593840642113284245e+00 1.800734326241278938e-01 -7.559136265638382035e-01 1.915135898433854011e+00 3.832675826531472651e+00 2.110202887357992108e+00 3.144831594687559573e+00 3.673976531312315164e-01 2.854609800849534196e+00 2.254043041949877146e+00 -7.039100689172012153e+02
