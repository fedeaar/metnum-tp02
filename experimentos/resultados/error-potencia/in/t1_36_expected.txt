-9.020000000000000000e+02
7.910000000000000000e+02
7.510000000000000000e+02
-7.340000000000000000e+02
7.090000000000000000e+02
-7.040000000000000000e+02
-6.980000000000000000e+02
-6.500000000000000000e+02
5.810000000000000000e+02
-5.520000000000000000e+02
5.270000000000000000e+02
-4.920000000000000000e+02
-4.080000000000000000e+02
-3.980000000000000000e+02
-3.160000000000000000e+02
-2.000000000000000000e+02
-1.160000000000000000e+02
-8.200000000000000000e+01
4.500000000000000000e+01
1.300000000000000000e+01
